cli/pcluster/config/json_param_types.py

```diff
@@ -82,6 +82,10 @@ def _parse_value(self, config_parser, section_name):
         # Convert value to boolean
         return config_parser.getboolean(section_name, self.key)
 
+    def get_string_value(self):
+        """Convert internal representation into string."""
+        return self.get_default_value() if self.value is None else str(bool(self.value)).lower()
```

Comment:
Do we not need `str(self.get_default_value()).lower()` here to keep things consistent?

Reply:
Done